core_backend/app/llm_call/llm_prompts.py

```diff
@@ -206,24 +204,50 @@ def get_prompt(cls) -> str:
 )
 
 
-# ----  Question answering bot
-ANSWER_FAILURE_MESSAGE = "Sorry, no relevant information found."
-ANSWER_QUESTION_PROMPT = f"""
-You are a question answering system that is constantly learning and improving.
-You can process and comprehend many pieces of text and utilize this knowledge to \
-provide concise, accurate, and informative answers to diverse queries.
+# ---- Response generation
+ANSWER_FAILURE_MESSAGE = "FAILED"
```

Comment:
There's no need for the failure message to be long! So I changed it

Reply:
Is this the responsed sent to the user or the response to the paraphrasing. If it is that later maybe "PARAPHRASHING FAILED"would be more intuitive

Reply:
We should prob clean up the terminology.. this is just for RAQ response generation failure. This is only used internally if response generation failed. @amiraliemami had some ideas to clean up actual error codes depending on where in the guardrails/RAG pipeline they come from